backend/src/test/java/de/neuefische/cgnjava222/zoomanagement/zoo/plant/PlantIntegrationTest.java

```diff
@@ -112,4 +113,59 @@ void deleteNoPlantExists() throws Exception {
         mockMvc.perform(delete("http://localhost:8080/api/plants/" + id))
                 .andExpect(status().is(404));
     }
+
+    @Test
+    @DirtiesContext
+    void getPlantById() throws Exception {
+
+        String saveResult = mockMvc.perform(post(
+                "/api/plants")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content("""
+                        {"name": "Pflanzen"}
+                        """)
+        ).andReturn().getResponse().getContentAsString();
+
+        Plant saveResultPlant = objectMapper.readValue(saveResult, Plant.class);
+        String id = saveResultPlant.id();
+
+        String teststring = objectMapper.writer().writeValueAsString(saveResultPlant);
+        System.out.println(teststring);
+        mockMvc
+                .perform(
+                        get("/api/plants/" + id)
+                )
+                .andExpect(status().isOk())
+                .andExpect(content().json(teststring));
```

Comment:
Mhhhh.... Ich weiß nicht genau, ob Ihr damit nicht einen Evergreen generiert, wenn Ihr erst den String erstellen lasst im gleichen Test, in dem Ihr dann die Existenz dieses Strings testet.... 🤔